### PR TITLE
Update the `prefer-power-assert` rule for AVA assert changes

### LINKED ISSUE #62: Update the `prefer-power-assert` rule for AVA assert changes
https://github.com/sindresorhus/ava/commit/e9c6cc2843e9ee6127aebb2843eafec86cb39bfc
https://github.com/sindresorhus/ava/commit/a7f50eb82d8dae4f9a7e70ae33ef71712a8fd69b

### PR DESCRIPTION
**Pull request by forresst**
Fixes #62

* rename `t.same()` to `t.deepEqual()`
* rename `t.notSame()` to `t.notDeepEqual()`
* rename `t.ok()` to `t.truthy()`
* rename `t.notOk()` to `t.falsy()`
* change bad link to "Enhanced assertion messages"

This is my first PR, be kind :smile:

### CODE REVIEW

File: docs/rules/prefer-power-assert.md [hunk 1]
```diff
@@ -28,6 +28,6 @@ test(t => {
 import test from 'ava';
 
 test(t => {
-	t.ok(foo === bar);
+	t.truthy(foo === bar);
```
Comment (sindresorhus) [inline docs/rules/prefer-power-assert.md]: @twada Do we want to continue using a truthy check or would it maybe be better to use `t.true` instead?

**Comment by jfmengels** [comment]
LGTM, thanks @forresst!

File: docs/rules/prefer-power-assert.md (hunk 1 above)
Comment (twada) [inline docs/rules/prefer-power-assert.md]: @sindresorhus I'm +1 to run with `t.truthy` ;)

**Comment by twada** [comment]
@forresst Looks very good! :+1:

**Comment by sindresorhus** [comment]
Excellent work @forresst :)

**Pull request merged**